src/React/Ref.purs

```diff
@@ -0,0 +1,57 @@
+module React.Ref
+  ( Ref
+  , RefHandler
+  , ReactInstance
+  , NativeNode
+  , fromRef
+  , fromEffect
+  , getCurrentRef
+  , createDOMRef
+  , createInstanceRef
+  ) where
+
+import Prelude
+import Effect (Effect)
+import Data.Maybe (Maybe)
+import Data.Nullable (Nullable)
+import Data.Nullable as Nullable
+import Effect.Uncurried (EffectFn1, runEffectFn1, mkEffectFn1)
+import Unsafe.Coerce (unsafeCoerce)
+
+--- | An instance of a React class.
+foreign import data ReactInstance :: Type
+
+--- | A platform-specific native layout node. On the web this will be a DOM
+--- | element (see `Web.HTML.HTMLElement`).
+foreign import data NativeNode :: Type
+
+foreign import data Ref :: Type -> Type
+
+foreign import data RefHandler :: Type -> Type
+
+
+foreign import createRef :: forall a. Effect (Ref a)
+
+foreign import liftCallbackRef :: forall a. Ref a -> Ref a
+
+
+createDOMRef :: Effect (Ref NativeNode)
```

Comment:
Small comment: should this be `createNodeRef` or something along those lines?

Reply:
Indeed it should!